index.html

```diff
@@ -296,6 +296,21 @@ <h3 id="praveen">Praveen Patil</h3>
                         </div>
                     </div>
                     <hr>
+                    <!-- Developer Three -->
+                    <div class="row">
+                        <div class="col-lg-4 col-sm-6 text-center">
+                            <img class="img-circle img-responsive img-center" src="img/asitavas.JPG" alt="">
+                            <h3>
+                                <small><a href="https://github.com/asitava1998">Asitava Sarkar</a></small>
+                            </h3>
+                        </div>
+                        <div class="col-lg-8">
+                            <h3 id="padmal">Asitava Sarkar</h3>
```

Comment:
Id should also change!

Reply:
I just copied the format verbatim from your PR. So, while changing, some things were missed out. Sorry for that.